Makefile

```diff
@@ -40,3 +41,8 @@ e2e: test-unit
 clean:
 	$(RM) ./cluster-kube-scheduler-operator
 .PHONY: clean
+
+# Configure the 'telepresence' target
+# See vendor/github.com/openshift/build-machinery-go/scripts/run-telepresence.sh for usage and configuration details
+export TP_DEPLOYMENT_YAML ?=./manifests/0000_25_kube-scheduler-operator_06_deployment.yaml
+export TP_CMD_PATH ?=./cmd/cluster-kube-scheduler-operator
```

Comment:
Since you don't use `$(call telepresence ...)`, will both variables have any effect?

Reply:
Thank you for your review! The make target added by line 10 will be configured by these values.  I've tested it to work on this and other PRs adding the telepresence target.  Feel free to give it a try! :)